examples/full/cmd/root.go

```diff
@@ -71,6 +71,10 @@ func (r *DaemonRunner) Run(ctx context.Context) error {
 	return RunServer(ctx, c)
 }
 
+const (
+	TeamDevelopment = `rebuy-tech`
```

Comment:
Not sure if we have this anywhere yet, but I don't think this fits in a "public" SDK. No hard feelings though.